backend/src/openbeheer/utils/open_zaak_helper/data_creation.py

```diff
@@ -40,6 +41,9 @@ def _create_resource[T](
         with ztc_client("OZ") as client:
             response = client.post(resource_path, json=data)
 
+            if response.status_code == 400:
+                raise Exception(decode(response.content))
```

Comment:
This immediately shows the "invalidParams" in the Exception, without having to catch it reference the RequestError.response.json().